Home brewed.
This is to fix info syntax for four e-mail addresses and add missing menu item.
Will be sent up-stream.

--- screen-4.9.1/doc/screen.texinfo.orig	2023-09-15 16:51:09.229857347 +0200
+++ screen-4.9.1/doc/screen.texinfo	2023-09-15 16:53:45.258028845 +0200
@@ -2416,6 +2416,7 @@
 * Monitor::                     Watch for activity or inactivity in a window
 * Windows::			List the active windows
 * Hardstatus::			Set a window's hardstatus line
+* Mousetrack::      Set whether screen should track mouse events.
 @end menu
 
 @node Naming Windows, Console,  , Window Settings
@@ -5869,10 +5870,10 @@
 ============
 
 @verbatim
-     Maarten ter Huurne <maarten@treewalker.org>,
-     Jussi Kukkonen <jussi.kukkonen@intel.com>,
-     Thomas Renninger <treen@suse.com>,
-     Axel Beckert <abe@deuxchevaux.org>,
+     Maarten ter Huurne <maarten@@treewalker.org>,
+     Jussi Kukkonen <jussi.kukkonen@@intel.com>,
+     Thomas Renninger <treen@@suse.com>,
+     Axel Beckert <abe@@deuxchevaux.org>,
      Ken Beal <kbeal@@amber.ssd.csd.harris.com>,
      Rudolf Koenig <rfkoenig@@informatik.uni-erlangen.de>,
      Toerless Eckert <eckert@@informatik.uni-erlangen.de>,
